diff --git a/node_modules/@react-native-community/cli-server-api/build/index.js b/node_modules/@react-native-community/cli-server-api/build/index.js
index 1afc60d..c2176f3 100644
--- a/node_modules/@react-native-community/cli-server-api/build/index.js
+++ b/node_modules/@react-native-community/cli-server-api/build/index.js
@@ -4,6 +4,7 @@ Object.defineProperty(exports, "__esModule", {
   value: true
 });
 exports.createDevServerMiddleware = createDevServerMiddleware;
+exports.indexPageMiddleware = void 0;
 function _compression() {
   const data = _interopRequireDefault(require("compression"));
   _compression = function () {
@@ -40,6 +41,7 @@ function _serveStatic() {
   return data;
 }
 var _indexPageMiddleware = _interopRequireDefault(require("./indexPageMiddleware"));
+exports.indexPageMiddleware = _indexPageMiddleware.default;
 var _openStackFrameMiddleware = _interopRequireDefault(require("./openStackFrameMiddleware"));
 var _openURLMiddleware = _interopRequireDefault(require("./openURLMiddleware"));
 var _securityHeadersMiddleware = _interopRequireDefault(require("./securityHeadersMiddleware"));
